frontend/src/screens/apps/NewApp.tsx

```diff
@@ -187,10 +185,6 @@ const NewAppInternal = ({ capabilities }: NewAppInternalProps) => {
 
     setLoading(true);
     try {
-      if (apps?.some((existingApp) => existingApp.name === appName)) {
```

Comment:
On the backend maybe we should automatically add a suffix e.g. `Alby Go (1)` if an app already exists with the same name

Reply:
done